twyla/chat/templates/image.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class ImagePayload:
+    url: str
+    is_reusable: bool = True
+
+
+@dataclass
+class ImageTemplate:
+    payload: ImagePayload
```

Comment:
Would be good for url and is_reusable to be InitVars here and for payload to be set only in a post init function.

Reply:
@abn so there shouldn't be a separate imagePayload class?

Reply:
A separate class yes, but not one that is exported so an end user does not need to be aware of it.